2022/day13/mastrogiovanni/parts.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func PartOne(inputs []string) string {
+	tot := 0
+	index := 1
+	for i := 0; i < len(inputs); i += 3 {
+		left := inputs[i]
+		right := inputs[i+1]
+		if res := Compare3(left, right); res {
+			tot += index
+		}
+		index++
+	}
+	return fmt.Sprintf("%d", tot)
+}
+
+func PartTwo(inputs []string) string {
+	list := make(Items, 0)
+	for i := 0; i < len(inputs); i += 3 {
+		left := parse(inputs[i])
+		right := parse(inputs[i+1])
+		list = append(list, left, right)
+	}
+	list = append(list, parse("[[2]]"), parse("[[6]]"))
+	sort.Sort(list)
```

Comment:
E' equivalente, ma ti consiglio di vedere `sort.Slice`.
Ti evita di creare un nuovo tipo e implementare 3 metodi.

Reply:
@lucianoq See if it's ok now. Thanks for your suggestion